samples/features/azure-arc-data-controller/deployment/kubeadm/ubuntu-single-node-vm/README.md

```diff
@@ -0,0 +1,47 @@
+
+# Deploy a Azure Arc Data Controller on single node Kubernetes cluster (kubeadm)
+
+Using this sample bash script, you will deploy a single node Kubernetes cluster using  kubeadm and a Azure Arc Data Controller on top of it. The script must be run from the VM you are planning to use for your kubeadm deployment.
+
+## Pre-requisites
+
+1. A vanilla Ubuntu 16.04 or 18.04 virtual or physical machine. All dependencies will be setup by the script. Using Azure Linux VMs is not yet supported.
+1. Machine should have at least 8 CPUs, 64GB RAM and 100GB disk space. After installing the images you will be left with 50GB for data/logs across all components.
```

Comment:
You have to evaluate the memory & disk requirements. This is the minimal version for BDC based tweaked config. This may not be sufficient for Azure Arc depending on how many resources are being created by default. So test & adjust accordingly.

Reply:
Currently this is the minimum we've tested the cluster with and it's sufficient.